tests/playwright/src/model/pages/mcp-page.ts

```diff
@@ -41,20 +42,35 @@ export class McpPage extends BasePage {
   }
 
   async openEditRegistriesTab(): Promise<McpEditRegistriesTabPage> {
-    await expect(this.editRegistriesTabButton).toBeEnabled();
-    await this.editRegistriesTabButton.click();
-
-    const editRegistriesTabPage = new McpEditRegistriesTabPage(this.page);
-    await editRegistriesTabPage.waitForLoad();
-    return editRegistriesTabPage;
+    return this.openTab(this.editRegistriesTabButton, McpEditRegistriesTabPage);
   }
 
   async openInstallTab(): Promise<McpInstallTabPage> {
-    await expect(this.installTabButton).toBeEnabled();
-    await this.installTabButton.click();
+    return this.openTab(this.installTabButton, McpInstallTabPage);
+  }
+
+  async openReadyTab(): Promise<McpReadyTabPage> {
+    return this.openTab(this.readyTabButton, McpReadyTabPage);
+  }
+
+  async createServer(serverName: string, token: string): Promise<void> {
+    const readyTab = await this.openReadyTab();
+
+    if (await readyTab.isServerConnected(serverName)) {
+      console.log(`MCP server ${serverName} is already connected, skipping...`);
+      return;
+    }
+
+    const installTab = await this.openInstallTab();
+    await installTab.installRemoteServer(serverName, token);
+
+    const readyTabAfterInstall = await this.openReadyTab();
+    expect(await readyTabAfterInstall.isServerConnected(serverName, 15_000)).toBe(true);
```

Comment:
In combination with the suggestion from the method itself this assert needs to be refactored, it should look like


```suggestion
    await expect.poll(async () => readyTabAfterInstall.isServerConnected(serverName, 15_000)).toBeTruthy();
```